INSTALL.md

```diff
@@ -94,20 +94,19 @@ permissions (like `system:admin`), as it needs to create a
 `ClusterRole` for deploying function containers from the `fission`
 namespace/project.
 
-Identically as with Kubernetes, you need to set the FISSION_URL and FISSION_ROUTER environment variables. If you're using minishift, use these commands:
-
-```
-  $ export FISSION_URL=http://$(minishift ip):31313¬
-  $ export FISSION_ROUTER=$(minishift ip):31314¬
-```
 #### Using Minishift or Local Cluster
```

Comment:
But that section is for "other clouds", as an alternative to minishift or a local cluster.  Maybe it should just be a new section, "CLI setup".